src/registry_schemas/example_data/schema_data.py

```diff
@@ -1210,6 +1210,12 @@
     'fileKey': '011e332d-1b8e-4218-8710-ad8ac1fbc592.pdf'
 }
 
+CONSENT_CONTINUATION_OUT = {
+    'orderDetails': 'A note to explain the consent to continue out',
+    'fileNumber': '#1234-5678/90',
+    'effectOfOrder': 'planOfArrangement'
+}
```

Comment:
What about `courtOrder`?

Reply:
`courtOrder` filing has a different required prop (`fileNumber`) which I cannot specify outside schema. Thats why we created separate schema

Reply:
Yeah! This is what I propose. Will wait for @thorwolpert feedback.